xgboost.ipynb

```diff
@@ -0,0 +1,227 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This example is similar to the example at https://dask-ml.readthedocs.io/en/latest/examples/xgboost.html, but this example takes a while to run because it uses a real dataset. There is a recorded screencast that walks through this example  at https://www.youtube.com/watch?v=Cc4E-PdDSro, with comments from the lead Dask developer, Matthew Rocklin.\n",
```

Comment:
I recommend that we start strong with the topic at hand, rather than discuss the other examples that are out there.  Maybe something like "Dask and XGBoost can work together to train data with gradient boosted trees.  This notebook shows how"

The context of other examples that may exist may not interest new users.  It's usually an accurate guess that for anyone looking at an example it is their first time seeing the topic.  If we do want to provide a link to the other example, lets do it at the bottom in a "Learn more" section

Reply:
I've expanded, and have included a paragraph about what XGBoost is and why it's useful, and it provides good motivation for the rest of the notebook. I left same thing for Dask out because if they're looking at this example they already want to learn about Dask. Let me know if you'd like me to change that.